src/tally.F90

```diff
@@ -92,7 +92,8 @@ subroutine score_general(p, t, start_index, filter_index, i_nuclide, &
           end if
           score = score / material_xs % total
 
-        else if (t % estimator == ESTIMATOR_TRACKLENGTH) then
+        else if (t % estimator == ESTIMATOR_TRACKLENGTH .or. &
+                 t % estimator == ESTIMATOR_COLLISION) then
```

Comment:
All of these "else if tracklength or collision" could probably just be "else". No comparisons needed, although it is at the expense of clarity.